### ISSUE TITLE
Indexing maps for ever on Android 9 - OsmAnd Live (Honor)

### ISSUE THREAD

**Issue by williamdes** [osmandapp/OsmAnd#11371]
### Description

![Screenshot_20210408_204959_com huawei android launcher](https://user-images.githubusercontent.com/7784660/114081165-83533f80-98ac-11eb-862c-b00131978712.jpg)

### How to reproduce?

When I let my phone auto update maps, afterwards a sticky notification runs endlessly. 
If I kill the app and start it back it does not do it anymore until next maps update.

### Your Environment
Premium subscription: yes
OsmAnd Version: latest
Android/iOS version: 9
Device model: Honor P9 lite

**Comment by vshcherb** [comment]
There is a similar issue for SDCard Android 11 - https://github.com/osmandapp/OsmAnd/issues/10453.
- But it's also possible that your issue is related that memory card is slow
- Another possible case that you use muliple maps with Live Updates i.e. 10-30 live regions could produce a lot of files (500 - 1 000) and it could slow down initialization.

**Comment by williamdes** [comment]
> But it's also possible that your issue is related that memory card is slow

Just did a benchmark and here are the results
![image](https://user-images.githubusercontent.com/7784660/114198310-76d5f200-9953-11eb-90b1-f3ddfab2dae6.png)

> Another possible case that you use muliple maps with Live Updates i.e. 10-30 live regions could produce a lot of files (500 - 1 000) and it could slow down initialization.

Yes, I have:
- France
- Switzerland
- Global world map (240.70MB - 2021)
- World altitude corrections (1.6MB - 2005)

A total of 5.36 GB on my SD card

**Comment by stephan75** [comment]
What is the name of that benchmark app?

**Comment by williamdes** [comment]
> What is the name of that benchmark app?

[A1 SD Bench](https://play.google.com/store/apps/details?id=com.a1dev.sdbench)

**Comment by vshcherb** [comment]
Workaround would be to disable live updates for certain regions and we will investigate how to compact number of files on sdcard

**Comment by uwemock** [comment]
I have a similar, possibly the same, issue on a Huawei MediaPad with Emui 9.1.0. When I update multiple files, usually the last one in row will be indexed forever.

Today, I installed the "Europe - Germany - Bavaria" standard map, and look what I got:

![2021-05-26_osmand](https://user-images.githubusercontent.com/31033449/119628986-e4f44c80-be0d-11eb-9dea-a934042dd8a8.jpg)

This file, Default_wikivoyage.travel.obf, was indexed "forever", and I had to force stop OsmAnd. Obviously, this i not the map that I had just installed.
